services-camel/src/main/java/edu/unc/lib/dl/services/camel/solr/CdrEventToSolrUpdateProcessor.java

```diff
@@ -86,25 +83,15 @@ public void process(Exchange exchange) throws Exception {
         List<String> subjects = populateList("subjects", contentBody);
 
         if (MOVE.equals(solrActionType)) {
-            offer(targetId, IndexingActionType.MOVE, subjects);
+            messageSender.sendIndexingOperation(userid, Collections.singletonList(PIDs.get(targetId)),
```

Comment:
I think this change is the cause of the ClassCastException.  Previously it was passing `subjects` in as the children, which we should probably keep doing